From 8c649ce3b49389d773380175aa20d0cba029948a Mon Sep 17 00:00:00 2001
Message-Id: <patch-v2-1.6-8c649ce3b49-20210826T121804Z-avarab@gmail.com>
In-Reply-To: <cover-v2-0.6-00000000000-20210826T121804Z-avarab@gmail.com>
References: <cover-0.6-00000000000-20210825T231400Z-avarab@gmail.com>
	<cover-v2-0.6-00000000000-20210826T121804Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 26 Aug 2021 01:04:58 +0200
Subject: [PATCH v2 1/6] tr2: remove NEEDSWORK comment for "non-procfs"
 implementations
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

I'm fairly sure that there is no way on Linux to inspect the process
tree without using procfs, any tool such as ps(1), top(1) etc. that
shows this sort of information ultimately looks the information up in
procfs.

So let's remove this comment added in 2f732bf15e6 (tr2: log parent
process name, 2021-07-21), it's setting us up for an impossible task.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 compat/linux/procinfo.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/compat/linux/procinfo.c b/compat/linux/procinfo.c
index 578fed4cd31..15a89676c7a 100644
--- a/compat/linux/procinfo.c
+++ b/compat/linux/procinfo.c
@@ -25,7 +25,6 @@ static void get_ancestry_names(struct strvec *names)
 	}
 
 	return;
-	/* NEEDSWORK: add non-procfs-linux implementations here */
 }
 
 void trace2_collect_process_info(enum trace2_process_info_reason reason)
-- 
2.33.0.733.ga72a4f1c2e1

